platforms/documentation/docs/src/docs/userguide/authoring-builds/introduction/build_lifecycle_intermediate.adoc

```diff
@@ -0,0 +1,116 @@
+// Copyright (C) 2024 Gradle, Inc.
+//
+// Licensed under the Creative Commons Attribution-Noncommercial-ShareAlike 4.0 International License.;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://creativecommons.org/licenses/by-nc-sa/4.0/
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[build_lifecycle]]
+= Gradle Build Lifecycle
+
+The build lifecycle is the sequence of phases Gradle executes to turn your build scripts into completed work, from initializing the build environment to configuring projects and finally executing tasks.
+
+image::gradle-basic-10.png[]
+
+== Build Phases
+
+A Gradle build has three distinct phases.
+
+image::author-gradle-1.png[]
+
+Gradle runs these phases in order:
+
+[cols="1,1,1", options="header"]
+|===
+|Phase 1. Initialization
+|Phase 2. Configuration
+|Phase 3. Execution
+
+| - Detects the *settings file* +
+- Creates a link:{groovyDslPath}/org.gradle.api.initialization.Settings.html[`*Settings*`] instance +
+- Evaluates the *settings file* to determine which projects (and included builds) make up the build +
+- Creates a link:{groovyDslPath}/org.gradle.api.Project.html[`*Project*`] instance *for every project*
+
+| - Evaluates the *build files of every project* participating in the build +
```

Comment:
Wondering if it's worth mentioning explicitly as a line item:
Task configurations are evaluated

Reply:
Added.